core/src/test/java/org/elasticsearch/index/query/AbstractQueryTestCase.java

```diff
@@ -205,6 +205,7 @@ public static void init() throws IOException {
                 .put("node.name", AbstractQueryTestCase.class.toString())
                 .put(Environment.PATH_HOME_SETTING.getKey(), createTempDir())
                 .put(ScriptService.SCRIPT_AUTO_RELOAD_ENABLED_SETTING.getKey(), false)
+                .put("script.inline", "true")
```

Comment:
Because we use inline scripts in a language that was previously sandboxed? Like some test script engine?